src/hooks/usePriceImpact.ts

```diff
@@ -49,3 +46,8 @@ export function toHumanReadablePercent(priceImpact: Percent): string {
   const number = parseFloat(priceImpact.multiply(-1)?.toFixed(2))
   return `${sign}${number}%`
 }
+
+export function formatPriceImpact(impact: PriceImpact | undefined): string {
```

Comment:
maybe copy the mobile version to be consistent? or we could move to conedison: https://github.com/Uniswap/mobile/blob/c4f3ab31b2b9422230e0e308c8b9db1ddf9125c6/src/utils/format.ts#L295

Reply:
updated to use conedison